packages/vue/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@microlink/vue",
+  "description": "Convert your links into beautiful previews",
+  "homepage": "https://microlink.io/docs/sdk/integrations/vue/",
+  "version": "5.0.0",
+  "main": "dist/microlink.common.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microlinkhq/sdk/tree/master/packages/vue"
+  },
+  "bugs": {
+    "url": "https://github.com/microlinkhq/sdk/issues"
+  },
+  "keywords": [
+    "data",
+    "extraction",
+    "link",
+    "microlink",
+    "preview",
+    "previsualization"
+  ],
+  "scripts": {
+    "dev": "start-storybook -p 6006",
+    "build": "vue-cli-service build --target lib --formats commonjs --name microlink ./src/index.js",
+    "lint": "vue-cli-service lint",
+    "build-storybook": "build-storybook"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@microlink/vanilla": "^5.0.0"
```

Comment:
@Kikobeats it actually needs `@microlink/vanilla@5.1.0` (or whatever this next release is), as it doesn't work with the _current_ Vanilla (because the 1st parameter passed to  `microlink()` needs to exist in the DOM).

Reply:
👍